content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -177,46 +209,102 @@ spec:
 
 ```
 
-The above template defines parameters for each job object using a list of
-python dicts (lines 1-4).  Then a for loop emits one job yaml object
-for each set of parameters (remaining lines).
-We take advantage of the fact that multiple yaml documents can be concatenated
-with the `---` separator (second to last line).
-.)  We can pipe the output directly to kubectl to
-create the objects.
+The above template defines two parameters for each Job object using a list of
+python dicts (lines 1-4).  Then a for loop emits one Job manifest for each
+set of parameters (remaining lines).
+
+This example relies on a feature of YAML. One YAML file can contain multiple
+documents (Kubernetes manifests, in this case), separated by `---` on a line
+by itself.
+You can pipe the output directly to `kubectl` to create the Job objects.
 
-You will need the jinja2 package if you do not already have it: `pip install --user jinja2`.
-Now, use this one-line python program to expand the template:
+Next, use this one-line Python program to expand the template:
 
 ```shell
 alias render_template='python -c "from jinja2 import Template; import sys; print(Template(sys.stdin.read()).render());"'
 ```
 
-
-
-The output can be saved to a file, like this:
+Use `render_template` to convert the parameters and template into a single
+YAML file containing Kubernetes manifests:
 
 ```shell
+# This requires the alias you defined earlier
 cat job.yaml.jinja2 | render_template > jobs.yaml
 ```
 
-Or sent directly to kubectl, like this:
+You can view `jobs.yaml` to verify that `render_template` is working OK.
+
```

Comment:
nit: Would remove `OK`.
View `jobs.yaml` to verify that the `render_template` script is working [correctly].